data_steward/cdr_cleaner/cleaning_rules/deid/explicit_identifier_suppression.py

```diff
@@ -0,0 +1,206 @@
+    {% for concept_field in concept_fields %}
+    LEFT JOIN `{{project_id}}.{{sandbox_dataset_id}}.{{lookup_table}}` AS s{{loop.index}}
+      ON ob.{{concept_field}} = s{{loop.index}}.concept_id 
+    {% endfor %}
+    WHERE COALESCE(
+    {% for concept_field in concept_fields %}
+        {% if loop.previtem is defined %}, {% else %}  {% endif %} s{{loop.index}}.concept_id
+    {% endfor %}) IS NOT NULL
+)
+""")
+
+SUPPRESS_EXPLICIT_IDENTIFIER_RECORDS = JINJA_ENV.from_string("""
+ DELETE FROM
+  `{{project_id}}.{{dataset_id}}.{{observation_table}}`
+WHERE
+observation_id 
+IN( SELECT
+    observation_id
+    FROM `{{project_id}}.{{sandbox_dataset_id}}.{{intermediary_table}}` )
+    """)
+
+
+class ExplicitIdentifierSuppression(BaseCleaningRule):
+    """
+    suppress all records associated with a PPI vocabulary explicit identifier associated PPI vocabulary
+    """
+
+    def __init__(self, project_id, dataset_id, sandbox_dataset_id):
+        """
+        Initialize the class with proper info.
+
+        Set the issue numbers, description and affected datasets.  As other
+        tickets may affect this SQL, append them to the list of Jira Issues.
+        DO NOT REMOVE ORIGINAL JIRA ISSUE NUMBERS!
+        """
+        desc = f'suppress all records associated with a PPI vocabulary explicit identifier associated PPI vocabulary'
+        super().__init__(issue_numbers=ISSUE_NUMBERS,
+                         description=desc,
+                         affected_datasets=[
+                             cdr_consts.CONTROLLED_TIER_DEID,
+                             cdr_consts.COMBINED
+                         ],
+                         affected_tables=OBSERVATION,
+                         project_id=project_id,
+                         dataset_id=dataset_id,
+                         sandbox_dataset_id=sandbox_dataset_id)
+
+    def get_query_specs(self, *args, **keyword_args):
+        """
+        Return a list of dictionary query specifications.
+
+        :return:  A list of dictionaries.  Each dictionary contains a
+            single query and a specification for how to execute that query.
+            The specifications are optional but the query is required.
+        """
+
+        lookup_table_creation_query = {
+            cdr_consts.QUERY:
+                LOOKUP_TABLE_CREATION_QUERY.render(
+                    project_id=self.project_id,
+                    sandbox_dataset=self.sandbox_dataset_id,
+                    dataset_id=self.dataset_id,
+                    lookup_table=EXPLICIT_IDENTIFIER_CONCEPTS)
+        }
+
+        sandbox_query = {
+            cdr_consts.QUERY:
+                SANDBOX_EXPLICIT_IDENTIFIER_RECORDS.render(
+                    project_id=self.project_id,
+                    dataset_id=self.dataset_id,
+                    sandbox_dataset_id=self.sandbox_dataset_id,
+                    observation_table=OBSERVATION,
+                    intermediary_table=self.get_sandbox_tablenames(),
```

Comment:
`self.get_sandbox_tablenames()` returns a list instead of a single item
```suggestion
                    intermediary_table=self.get_sandbox_tablenames()[0],
```